docs/module_docs/requirements.txt

```diff
@@ -0,0 +1 @@
+sphinx-autoapi
```

Comment:
Does this need to be explicitly installed or does sphinx handle automatically?

Reply:
This is used by ReadTheDocs -- if you look at the `readthedocs.yml` file towards the end, the `requirements` key points to this file to install the listed packages before building the docs.

Given that I don't expect people to typically be building the docs locally and in the interest of fewer `environment.yml` dependencies, I did not include it there.